api/cluster/resource.go

```diff
@@ -165,6 +181,67 @@ func createPredictorSpec(modelService *models.Service, config *config.Deployment
 	return predictorSpec
 }
 
+func createTransformerSpec(modelService *models.Service, transformer *models.Transformer, config *config.DeploymentConfig) *kfsv1alpha2.TransformerSpec {
```

Comment:
Can we also specify target concurrency of the transformer?
The reason is that many of transformer will contain IO code (fetch from feast, db, etc) and it's wasteful to have target concurrency = 1.

Reply:
Unfortunately, we cannot do it at this moment. The annotation for configuring the target concurrency is specified in InferenceService that will be inherited by both Predictor and Transformer.

Reply:
Let's keep this in mind, especially when upgrading the KFserving